gradle.properties

```diff
@@ -3,32 +3,38 @@
 
 pluginGroup = com.github.brcosta.cljstuffplugin
 pluginName = clj-extras-plugin
-pluginVersion = 0.7.13
+pluginVersion = 0.8.0
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild = 203
+pluginSinceBuild = 242
 pluginUntilBuild = 242.*
 
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions.
-pluginVerifierIdeVersions = 2020.3.4, 2021.1.3, 2021.2.4, 2021.3.3, 2022.1, 2022.2, 2022.3, 2023.1, 2023.2, 2023.3, 2024.1, 2024.2
+pluginVerifierIdeVersions = 2024.2
```

Comment:
📓 @brcosta This is probably the biggest and most controversial change, but I think it's necessary. Colin long ago made the choice that new versions of Cursive will only run against the version of IJ they're built for, and clj-extras is a sort of sidecar to Cursive.

I initially tried reducing the list of supported versions down to 2022.3 and newer, but even then failed verifying the plugin because Cursive's underlying API had breaking changes. To get it to work correctly for the latest IJ/Cursive, changes were necessary. Once I'd started down that path, doing single-version lock-in seemed more sensible.